src/index.html

```diff
@@ -95,9 +95,9 @@
       <a class="navbar__nav-item navbar__link" href="#sponsor">
         Sponsors
       </a>
-      <!-- <a class="navbar__nav-item navbar__link" href="">
+      <a class="navbar__nav-item navbar__link" href="#faq">
```

Comment:
`#faq` is before `#sponsors` in how you ordered it on the page, but is after `#sponsors` in the navbar. Not sure which order it's going to be yet, pending design team. For now, keep them consistent.

Reply:
Update: Design team decided on FAQ at the bottom after sponsors; can we fix the order of this?